account_invoice_partner/README.rst

```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=================================================
+Automatically select invoicing partner on invoice
+=================================================
+
+On an invoice, when selecting a partner of any other type than 'invoice',
+replace the partner by an invoice contact if found.
+
+Installation
+============
+
+There are no specific installation instructions for this module.
+
+Configuration
+=============
+
+There are no specific configuration instructions for this module.
+
+Usage
+=====
+
+This module need base account module
```

Comment:
@GillesTephaneMeyomesse Fill this section (see https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst)